app/models/user/VersionTable.scala

```diff
@@ -33,6 +33,23 @@ object VersionTable {
     versions.list
   }
 
+  /**
+    * Returns current version ID
+    *
+    */
+  def currentVersionId(): String = db.withSession { implicit session =>
+    versions.sortBy(_.versionStartTime.desc).list.head.versionId
+  }
+
+  /**
+    * Returns date of most recent update
+    *
+    */
+  def currentVersionDate(): String = db.withSession { implicit session =>
```

Comment:
These two functions have replaced the `currentVersionStr` function, right? If `currentVersionStr` isn't used anywhere, can you remove it?